### PR TITLE
Isops 508

### PR DESCRIPTION
**Pull request by is-ckothapally**

### CODE REVIEW

**Review by is-srudra — DISMISSED** [review]

**Review by is-ttang — CHANGES_REQUESTED** [review]
need to assume 
1) container has been built because container build may need args
2) caller has pull down code and run other testing, build container, just need to publish - tag and push

**Comment by is-ckothapally** [comment]
> need to assume
> 
> 1. container has been built because container build may need args
> 2. caller has pull down code and run other testing, build container, just need to publish - tag and push

**Comment by is-ckothapally** [comment]
@is-ttang the actions have been changed without build arg dependency and can be called from caller repo.

File: actions/aws-login/action.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Setup AWS OIDC IAM Role via sharableaction
+
+inputs:
+  AWS_REGION:
+    required: true
+    type: string
+  ROLE_ARN:
+    required: true
+    type: string
+  SECRET_ID:
```
Comment (is-ttang) [inline actions/aws-login/action.yml]: could you remove SECRET_ID, which is not used. Second, to use lower case for inputs

File: actions/ecr-push/action.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+name: 'Push to ECR'
```
Comment (is-ttang) [inline actions/ecr-push/action.yml]: do we need this action? build-ecr-image should take care

**Review by is-ttang — APPROVED** [review]

**Pull request merged**